src/pudl/analysis/ferc1_eia.py

```diff
@@ -0,0 +1,1092 @@
+"""Connect FERC1 plants tables to EIA's plant-parts via record linkage.
+
+FERC plant records are reported... kind of messily. In the same table there are
+records that are reported as whole plants, generators, collections of prime
+movers. So we have this heterogeneously reported collection of parts of plants
+in FERC1.
+
+EIA on the other hand is reported in a much cleaner way. The are generators
+with ids and plants with ids reported in *seperate* tables. What a joy. In
+`pudl.analysis.plant_parts_eia`, we've generated the EIA plant-parts. The EIA
```

Comment:
The references to modules, methods, and functions in this docstring can be made into links within the documentation by using e.g.

```
:mod:`pudl.analysis.plant_parts_eia`
:meth:`Features.make_features`
:func:`run_model`
```

Strings to be `preformatted` need to be enclosed with double-backticks.

Reply:
ty!! these were old docs migrated over from rmi land that hadn't previously been actually building read the docs docs. i went through the module and found other instances of this & cleaned it up